modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,355 @@
+          ExecCondition=/usr/bin/test ! -d /var/lib/etcd/member
+          ExecStart=/usr/sbin/setenforce 0
+          ExecStart=/bin/rsync -ar /sysroot/ostree/deploy/rhcos/var/lib/etcd/ /var/lib/etcd/
+          ExecStart=/usr/sbin/setenforce 1
+          TimeoutSec=0
+
+          [Install]
+          WantedBy=multi-user.target graphical.target
+        enabled: true
+        name: sync-var-lib-etcd-to-etcd.service
+      - contents: |
+          [Unit]
+          Description=Restore recursive SELinux security contexts
+          DefaultDependencies=no
+          After=var-lib-etcd.mount
+          Before=crio.service
+
+          [Service]
+          Type=oneshot
+          RemainAfterExit=yes
+          ExecStart=/sbin/restorecon -R /var/lib/etcd/
+          TimeoutSec=0
+
+          [Install]
+          WantedBy=multi-user.target graphical.target
+        enabled: true
+        name: restorecon-var-lib-etcd.service
+----
+====
+
+. Create the new `MachineConfig` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f 98-var-lib-etcd.yaml
+----
++
+[NOTE]
+====
+It will take some time for the etcd to move onto the local disk of each control plane machine.
+====
+
+. Verify that the `etcd` has been moved to the local disk of each control plane by running the following commands:
++
+.. Verify that the cluster is still updating by running the following command:
++
+[source,terminal]
+----
+$ oc wait --timeout=45m --for=condition=Updating=false machineconfigpool/master
+----
+.. Verify that the cluster is ready by running the following command:
++
+[source,terminal]
+----
+$ oc wait node --selector='node-role.kubernetes.io/master' --for condition=Ready --timeout=30s
+----
+.. Verify that the `clusteroperators` are running in the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc wait clusteroperators --timeout=30m --all --for=condition=Progressing=false
+----
+
+. Create a file named `etcd-replace.yaml` by using the following content:
++
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-var-lib-etcd
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - contents: |
+          [Unit]
+          Description=Mount /dev/vdb to /var/lib/etcd
+          Before=local-fs.target
+          Requires=systemd-mkfs@dev-vdb.service
+          After=systemd-mkfs@dev-vdb.service var.mount
+
+          [Mount]
+          What=/dev/vdb
+          Where=/var/lib/etcd
+          Type=xfs
+          Options=defaults,prjquota
+
+          [Install]
+          WantedBy=local-fs.target
+        enabled: true
+        name: var-lib-etcd.mount
+----
+====
+
+. Remove the logic for creating and syncing the device by running the following command:
++
+[source,terminal]
+----
+$ oc replace -f etcd-replace.yaml
+----
++
```

Comment:
This whole section isn't required if we fix the previous section to be idempotent.